src/commands/run.rs

```diff
@@ -383,6 +383,12 @@ pub fn setup_run_dir(root: &Path, entrypoint: &str) -> Result<PathBuf> {
     std::fs::create_dir_all(&root)
         .with_context(|| format!("failed to create directory: `{dir}`", dir = root.display()))?;
 
+    let ignore_path = root.join(".sprocketignore");
```

Comment:
this should use the `const` defined in `lib.rs` - https://github.com/stjude-rust-labs/sprocket/blob/main/src/lib.rs#L40